README.md

```diff
@@ -1 +1,43 @@
-# covid_state_checker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/89effbe5-ea91-4f9d-a00a-8fc5d63b390d/deploy-status)](https://app.netlify.com/sites/covid-state/deploys)
+# Space Traveler Hub
+This web app allows users to see available rockets for booking and missions they can join. Users can see their booked rockets and joined missions in the My-profile page and cancel any joined mission or booked rocket.
+
+
+# Built with
+- React.js
+- Redux
+- Tailwind
+
+## Live Demo
+https://covid-state.netlify.app
+
+## installation
+
+To get a local copy up and running follow these simple example steps.
+
+- clone the repository by running
+``` git@github.com:kosher9/covid_state_checker.git ```
+- navigate to the folder
+``` cd <DIRECTORY> ```
+- Install packages
+``` npm i ```
+- Run the app
+``` npm start ```
+
+# Author
+
+👤 **Cedric Kossi**
+
+- GitHub: [@kosher9](https://github.com/kosher9)
+- Twitter: [@kosherus](https://twitter.com/kosherus)
+- LinkedIn: [Lionel Cedric KOSSI](https://linkedin.com/in/lionel-c%C3%A9dric-kossi-323042172)
+
+## :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+Give a :star:️ if you like this project!
+## Acknowledgments
+
+- Microverse offered guidance on how to make the project more professional
```

Comment:
- [ ] Let's make sure that the "Acknowledgements" section gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Well resolved 👍🏽